universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -75,6 +77,19 @@ public Content renderEditQuestionForm(Request request, QuestionDefinition questi
     return layout.renderFull(mainContent);
   }
 
+  public Content renderViewQuestionForm(Request request, QuestionDefinition question) {
+    QuestionForm questionForm = new QuestionForm(question);
+    QuestionType questionType = question.getQuestionType();
+    String title = String.format("Edit %s question", questionType.toString().toLowerCase());
+
+    ContainerTag formContent =
+        buildQuestionContainer(title).with(buildQuestionForm(questionForm, false));
+    ContainerTag previewContent = buildPreviewContent(questionType);
+    ContainerTag mainContent = main(formContent, previewContent);
+
+    return layout.renderFull(mainContent);
+  }
+
```

Comment:
Nit (non-blocking): please move this function one function below so it's not in between two versions of the same overloaded function.

Reply:
Done.